tx.go

```diff
@@ -204,11 +197,20 @@ func (tx *Tx) Commit() (err error) {
 		countFlag = CountFlagDisabled
 	}
 
-	buff := cachePool.Get().(*bytes.Buffer)
-	defer func() {
-		buff.Reset()
-		cachePool.Put(buff)
-	}()
+	var txSize int64
+	for i := 0; i < writesLen; i++ {
+		txSize += tx.pendingWrites[i].Size()
```

Comment:
This calculation logic can be warped into a method in tx obj. That will be more elegant.

Reply:
Okay.